packages/building-rollup/src/utils.js

```diff
@@ -39,11 +40,30 @@ function pluginWithOptions(plugin, userConfig, defaultConfig, ...otherParams) {
   return plugin(config, ...otherParams);
 }
 
+/**
+ *
+ * @param {SpaOptions} userOptions
+ * @param {string} outputDir
+ * @param {string} htmlFileName
+ */
+function createSwPath(userOptions, outputDir, htmlFileName = '') {
```

Comment:
Does `htmlFileName` need to have a default value? Isn't this always a known value?

Reply:
indeed - changed